javascript/index.js

```diff
@@ -14,52 +14,91 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
-}
 
+  printMinutes();
+  printSeconds();
+
+}
 function printMinutes() {
-  // ... your code goes here
+  const mins = chronometer.twoDigitsNumber(chronometer.getMinutes());
+    minDec.innerText = mins[0];
+    minUni.innerHTML = mins[1];
 }
 
 function printSeconds() {
-  // ... your code goes here
+  const secs = chronometer.twoDigitsNumber(chronometer.getSeconds());
+  secDec.innerText = secs[0];
+  secUni.innerHTML = secs[1];
 }
 
 // ==> BONUS
 function printMilliseconds() {
   // ... your code goes here
 }
 
-function printSplit() {
-  // ... your code goes here
-}
+  function printSplit() {    
+    let splitsCount = document.getElementById('splits')  
+    newSplit = document.createElement('li')  
+    newSplit.innerHTML = chronometer.splitClick()  
+    splitsCount.appendChild(newSplit)}
+
 
 function clearSplits() {
-  // ... your code goes here
+  document.getElementById("splits").innerHTML = "";
 }
 
 function setStopBtn() {
-  // ... your code goes here
+    btnLeft.classList.remove("stop");
+    btnLeft.classList.add("start");
+    btnLeft.innerHTML = "START";
 }
 
 function setSplitBtn() {
-  // ... your code goes here
+    btnRight.classList.remove('reset');
+    btnRight.classList.add('split');
+    btnRight.innerHTML = 'SPLIT';
 }
 
 function setStartBtn() {
-  // ... your code goes here
```

Comment:
Same comment as above, either rename this function or switch its logic with function setStopBtn